app/Listeners/RunForSavedForm.php

```diff
@@ -0,0 +1,21 @@
+<?php namespace App\Listeners;
+
+use Anomaly\SettingsModule\Setting\Form\SettingFormRepository;
+use Anomaly\Streams\Platform\Ui\Form\Event\FormWasSaved;
+
+class RunForSavedForm
+{
+
+    public function handle(FormWasSaved $event)
+    {
+        $builder = $event->getBuilder();
+
+        if (get_class($builder->getRepository()) === SettingFormRepository::class) {
+            if ($builder->getFormValues()->has('maintenance') and $builder->getFormValues()->get('maintenance')) {
+                file_put_contents(storage_path('framework/down'),
+                    json_encode(['time' => '', 'retry' => null, 'message' => null],
+                        JSON_PRETTY_PRINT));
+            }
```

Comment:
If maintenance mode is not changed, will this code run? it  looks like it will run even if the maintenance mode is not changed, so it will keep creating the down file each time the setting form is saved @fatihalp I am not sure if that's the case,  maybe its ok since we don't use maintenance mode that often

Reply:
+ Why dont you run the artisan command from here instead of duplicating the code?

Reply:
https://github.com/openclassify/openclassify/pull/877#discussion_r548458184

Reply:
`$builder->getFormValues()->get('maintenance')`